sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/io/BigQueryIO.java

```diff
@@ -975,18 +990,20 @@ public PDone apply(PCollection<TableRow> input) {
         String tempLocation = options.getTempLocation();
```

Comment:
how will this work when running on a system where options.getTempLocation() is not a GCS path?

One guess:

* Add GcsOptions.getTempGcsLocation()
* If options.getTempGcsLocation() is present, use it.
    * If not, fall back to options.getTempLocation().
* If not a gs:// path, throw an error with a good message.